build.gradle

```diff
@@ -17,7 +17,7 @@ import java.text.SimpleDateFormat
 
 plugins {
   id 'java-test-fixtures'
-  id 'com.diffplug.gradle.spotless' version '3.27.1'
+  id "com.diffplug.spotless" version "5.15.0"
```

Comment:
nit: why the double quotes? other dependencies are using single quotes

Reply:
ah, yep, copy paste :)